data/blog/governance.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Electron Governance
+author:
+ - ckerr
+ - sofianguy
+date: '2019-03-18'
+---
+
+As Electron grows in popularity for desktop applications, the team working on it has also grown: we have more fulltime maintainers who work for different companies, live in different timezones, and have different interests. We're introducing a governance structure so we can keep growing smoothly.
+
+---
+
+## Why are things changing?
+
+People in the Electron project coordinate in timezones around the world with volunteers, with fulltime maintainers, and with several companies who all rely on Electron. Until now we've been successful with informal coordination; but as the team grows, that approach doesn't scale. We also want to make it easier for new contributors to find a place to call home in the project.
+
+## Working Groups
+
+Electron governance includes working groups that are responsible for different parts of the project. We're starting out with seven groups:
+ * Community & Safety: Handles [Code of Conduct](https://github.com/electron/governance/blob/master/CODE_OF_CONDUCT.md) issues.
+ * Docs & Tooling: Oversees externally-focused tooling (e.g. [Fiddle](https://electronjs.org/fiddle), [Forge](https://electronforge.io/)) and the Electron [documentation](https://electronjs.org/docs).
+ * Outreach: Helps grow the Electron community.
+ * Releases: Ensures releases are stable and on schedule.
+ * Security: Performs security testing and responds to security issues.
+ * Upgrades: Integrates upstream upgrades, such as new versions of V8, Chromium, and Node.
+ * Website: You're lookin' at it. https://electronjs.org/
```

Comment:
Normally I like this kind of line in a post, but I wonder if it's not quite the right tone for this post. (Also, folks might be reading this in a feed reader, etc).

Reply:
Fixed @ 641f02c7028d676e24e71ac14796adb5b4aef1cc